.changes/v2.21.0/572-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Changed Org enablement status during tests for VCD 10.4.2, to circumvent a VCD bug that prevents creation of disabled Orgs [GH-572]
```

Comment:
That is a test only patch - should we keep it under "notes"?

Reply:
Changed